web/src/user_status_ui.ts

```diff
@@ -48,6 +132,28 @@ export function open_user_status_modal(): void {
             input_field().trigger("focus");
         },
     });
+
+    render_scheduled_end_time_widget();
+
+    $("#scheduled-end-time-description").text(no_scheduled_end_time_set_message);
+    if (scheduled_end_time !== undefined) {
+        $("#scheduled_end_time_widget .dropdown_widget_value").text(
+            $t(
+                {defaultMessage: "{N}"},
+                {
+                    N: timerender.get_full_datetime(new Date(scheduled_end_time), "time"),
+                },
+            ),
+        );
+        $("#scheduled-end-time-description").toggleClass("invisible", true);
```

Comment:
We mostly use `.show`/`.hide` or toggle `hide` class for showing and hiding elements. Can we do that here as well?